robottelo/config/base.py

```diff
@@ -1620,6 +1620,21 @@ def _configure_third_party_logging(self):
         for logger in loggers:
             logging.getLogger(logger).setLevel(logging.WARNING)
 
+    def to_dict(self):
```

Comment:
we do have some nested values that might trip this up since it isn't recursive.

Reply:
Bruno's idea was to introduce properties-to-yaml translation tool, and that method was needed for that tool. Since I don't intent to implement such tool (I introduce facade so old new settings can co-exist and we can port them gradually), I simply dropped this method.